optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/tenant/TemplateEditor.java

```diff
@@ -69,37 +68,36 @@
-                .withBeanManager(beanManager)
-                .creatingDataInstancesWith((c, name, start, end) -> {
-                    Shift newShift = new Shift(getTenantId(),
-                            name.getSpot(),
-                            new TimeSlot(getTenantId(),
-                                    start,
-                                    end));
-                    newShift.setId(templateFetchable.getFreshId());
-                    c
-                            .addShift(new ShiftData(new SpotData(newShift)));
-                })
-                .asTwoDayView((v, d, i) -> new ShiftDrawable(v, d, i));
+                                                                                                              .fetchingDataFrom(templateFetchable)
+                                                                                                              .fetchingGroupsFrom(new SpotNameFetchable(() -> getTenantId()))
+                                                                                                              .displayWeekAs(DateDisplay.WEEKS_FROM_EPOCH)
+                                                                                                              .withBeanManager(beanManager)
+                                                                                                              .creatingDataInstancesWith((c, name, start, end) -> {
+                                                                                                                  Shift newShift = new Shift(getTenantId(),
+                                                                                                                                             name.getSpot(),
+                                                                                                                                             new TimeSlot(getTenantId(),
+                                                                                                                                                          start,
+                                                                                                                                                          end));
+                                                                                                                  newShift.setId(templateFetchable.getFreshId());
+                                                                                                                  c
+                                                                                                                   .addShift(new ShiftData(new SpotData(newShift)));
+                                                                                                              })
+                                                                                                              .asTwoDayView((v, d, i) -> new ShiftDrawable(v, d, i));
```

Comment:
@Christopher-Chianelli Are you sure you're using the latest eclipse formatter file in your Eclipse IDE?
The format files are here:
  https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration/eclipse-configuration/code-style
The README of droolsjbpm-build-bootstrap explains how to set them in Eclipse.

This is not blocking the merge, so I 'll merge.

@hasys There seems to be 2 different sets for the eclipse formatter. Is that correct? (If it is, one must be removed - there can only be one.)
https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration/eclipse-configuration/code-style

Reply:
Hi @ge0ffrey,
no, it is correct. Eclipse has 3 files instead of one [1] `Clean Up` file, `Formatter` and `Organize Imports`. Honestly, it is support 4-th file as well - `Code Templates`, but it is not predefined by KIE Group formatter.

[1]
![image](https://user-images.githubusercontent.com/1477262/36894258-112dba8a-1e0b-11e8-81e0-8201f5ed8dac.png)

Reply:
Yes, @Christopher-Chianelli sorry it is described little bit confusing:

`Use the one from droolsjbpm-build-bootstrap/ide-configuration`

It is going from precondition that Eclipse users know how does it work.
@Christopher-Chianelli, you are right, it is better to describe better that all tree files are needed. Can I ask you to fix this in [README file here](https://github.com/kiegroup/droolsjbpm-build-bootstrap)?

Also: @ge0ffrey @Christopher-Chianelli Eclipse and Idea formatters are not 100% equivalent, but very close to it. I tired to align it as possible, some staff is working just differently :( I guess it is an example of differences. If you are able to find out a way how to make it better it will be cool.